### PR TITLE
Allow `str` for `tkinter.Scrollbar.set()`

### LINKED ISSUE #10071: tkinter scrollbar set accept also str not only float
Tkinter scrollbar set() accepts also str not only float:

    def set(self, first: float, last: float) -> None: ...

should be:

    def set(self, first: float|str, last: float|str) -> None: ...

### PR DESCRIPTION
**Pull request by Paulie-Aditya**
Fixes #10071

### CODE REVIEW

File: stdlib/tkinter/__init__.pyi [hunk 1]
```diff
@@ -2888,7 +2888,7 @@ class Scrollbar(Widget):
     def fraction(self, x: int, y: int) -> float: ...
     def identify(self, x: int, y: int) -> Literal["arrow1", "arrow2", "slider", "trough1", "trough2", ""]: ...
     def get(self) -> tuple[float, float, float, float] | tuple[float, float]: ...
-    def set(self, first: float, last: float) -> None: ...
+    def set(self, float(first), float(last)) -> None: ...
```
Comment (AlexWaygood) [inline stdlib/tkinter/__init__.pyi:2891]: Hmm, this isn't valid Python syntax :)

Are you sure you fully understand the issue you're trying to fix?

Comment (Paulie-Aditya) [reply stdlib/tkinter/__init__.pyi:2891]: yeah just a sec fixing it now

Comment (AlexWaygood) [reply stdlib/tkinter/__init__.pyi]: No, that wouldn't work. Functions in stub files shouldn't have any bodies.

If you're new to the idea of what a stub file is, please see https://peps.python.org/pep-0484/#stub-files or https://typing.readthedocs.io/en/latest/source/stubs.html#type-stubs for an explanation :)

**Review by Akuli — APPROVED** [review]
Thanks!

**Pull request merged**